indexer-app/src/cmr/indexer/data/concepts/granule.clj

```diff
@@ -209,6 +213,8 @@
             :attributes (attrib/psa-refs->elastic-docs parent-collection umm-granule)
             :revision-date revision-date
             :revision-date-doc-values revision-date
+            :revision-date-stored revision-date
```

Comment:
Only add one new value - maybe `revision-date-stored-doc-values`.

Reply:
In search-app, we have a config param to search on doc-values vs non doc-values. Is that code obsolete?

Reply:
Yes, we should delete that code.